tests/system/memory_leak/test_memcheck.sh

```diff
@@ -116,7 +118,11 @@ generate_valgrind_logs(){
   echo 'Creating reports directory';
   mkdir -p reports/test1 ; ls -lrth
```

Comment:
Not sure why are we creating reports directory with **test1**? What does it really mean?